modules/minimp3/audio_stream_mp3.cpp

```diff
@@ -314,7 +315,12 @@ Ref<AudioSample> AudioStreamMP3::generate_sample() const {
 	return sample;
 }
 
+Ref<AudioStreamMP3> AudioStreamMP3::load_from_file(const String &p_path) {
+	return ResourceImporterMP3::import_mp3(p_path);
+}
+
 void AudioStreamMP3::_bind_methods() {
+	ClassDB::bind_static_method("AudioStreamMP3", D_METHOD("load_from_file", "path"), &AudioStreamMP3::load_from_file);
```

Comment:
Just add a little space between static and other methods.

```suggestion
	ClassDB::bind_static_method("AudioStreamMP3", D_METHOD("load_from_file", "path"), &AudioStreamMP3::load_from_file);
	
```